examples/large_models/inferentia2/llama2/requirements.txt

```diff
@@ -1,3 +1,3 @@
-transformers==4.31.0
+transformers==4.36.0
```

Comment:
This is resolved here: https://github.com/pytorch/serve/pull/2893/files#diff-c8949a67739ac8a79e2d88242b21347e2bc29a5ef4747cddd4a03b2aca28a126R1